### ISSUE TITLE
When viewing wager detail page, show more detailed information

### ISSUE THREAD

**Issue by ernieturner** [cwenzel/sharpgroups#8]
If wager has been processed, it should show how much was won/lost
If wager has not been processed, it should show how much could be won

**Comment by cwenzel** [comment]
Yep, this is a good one. If whoever works this does any look/feel changes just try to emulate the little slip that you get from an actual sportsbook.

**Comment by kevinfenger** [comment]
I'm kind of working on this, with the boarditem ui rewrite.
